packages/central-server/src/database/constants.js

```diff
@@ -4,3 +4,5 @@
  */
 
 export const DEFAULT_DATABASE_TIMEZONE = 'Pacific/Auckland';
+
+export const CONFIG_BREAK_MEDITRAK_APP_VERSION = '1.13.127';
```

Comment:
```suggestion
export const SURVEY_SCREEN_COMPONENT_ENTITY_CONFIG_SCHEMA_CHANGE_VERSION = '1.13.127';
```
Bit of a mouthful so not too set on it, but it'd be nice to have a descriptive name for this.

Also, I think we might need to bump the meditrak app version as part of this